op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -78,45 +81,43 @@ var (
 		Usage: "Memory limit in MiB, should be set lower than the available amount of memory in your system to prevent out of memory errors",
 		Value: 7500,
 	}
-	clearAllFlag = &cli.BoolFlag{
-		Name:  "clear-all",
-		Usage: "Use this to start with a fresh new db, deleting all data including ancients. CAUTION: Re-migrating ancients takes time.",
+	clearNonAncientsFlag = &cli.BoolFlag{
+		Name:  "clear-non-ancients",
+		Usage: "Use this to reset all data except ancients. This flag should be used if a full migration has already been performed on the new db.",
 	}
-	onlyAncientsFlag = &cli.BoolFlag{
-		Name:  "only-ancients",
-		Usage: "Use to only migrate ancient blocks. Ignored when running full migration",
+	measureTimeFlag = &cli.BoolFlag{
```

Comment:
I would remove this flag. It's useful to measure the time and it doesn't really have any cost, so we should just always do it.